buildSrc/src/main/kotlin/smithy-java.module-conventions.gradle.kts

```diff
@@ -49,6 +49,7 @@ afterEvaluate {
         manifest {
             attributes(mapOf("Automatic-Module-Name" to moduleName))
         }
+        duplicatesStrategy = DuplicatesStrategy.FAIL
```

Comment:
Isn't this the default?

Reply:
Default is `INHERIT`, which effectively fails if nothing is set AFAIK. We probably don't _need_ to set this, but better safe than sorry.